front/src/Phaser/Game/GameMapPropertyChange.ts

```diff
@@ -0,0 +1,55 @@
+import type { GameScene } from "./GameScene";
+import type { GameMap } from "./GameMap";
+import { scriptUtils } from "../../Api/ScriptUtils";
+import { coWebsiteManager } from "../../WebRtc/CoWebsiteManager";
+import { layoutManagerActionStore } from "../../Stores/LayoutManagerStore";
+import {
+    ON_ACTION_TRIGGER_BUTTON,
+    TRIGGER_WEBSITE_PROPERTIES,
+    WEBSITE_MESSAGE_PROPERTIES,
+} from "../../WebRtc/LayoutManager";
+
+export class GameMapPropertyChange {
+    constructor(private scene: GameScene, private gameMap: GameMap) {}
+
+    register() {
+        this.gameMap.onPropertyChange("openTab", (newValue) => {
+            if (typeof newValue == "string" && newValue.length) {
+                scriptUtils.openTab(newValue);
+            }
+        });
+        this.gameMap.onPropertyChange("openWebsite", (newValue, oldValue, allProps) => {
```

Comment:
just to be sure, you didn't edit this.gameMap.onPropertyChange("openWebsite", (...)) ?
You simply moved it away from GameScene?

Reply:
well its is slightly adjusted - i converted 2 `this.` to `this.scene.`